core/src/main/java/org/infinispan/persistence/manager/PersistenceManagerImpl.java

```diff
@@ -160,6 +164,19 @@
       return null;
    }
 
+   @GuardedBy("lock#readLock")
+   private StoreStatus getStoreStatusLocked(Predicate<? super StoreConfiguration> predicate) {
+      StoreStatus firstStoreStatus = null;
+      for (StoreStatus storeStatus : stores) {
+         if (storeStatus.characteristics.contains(Characteristic.BULK_READ) &&
```

Comment:
a comment saying why do we need `BULK_READ`.

Reply:
It shouldn't have been here, I extracted the method and forgot to compose the predicates in the caller